File: ex4.cpp
#include <iostream>

using namespace std;

int main() {
    int nota;
    cout << "Nota: " << endl;
    cin >> nota;
    if(nota >= 6) {
        cout << "Aprovado" << endl;
    } else {
        cout << "Reprovado" << endl;
    }

    cin >> nota;

    return 0;
}

File: aula5/Ex4.java
import java.util.Scanner;

class Ex4 {
    public static void main(String[] args) {
        float nota;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Digite sua nota: ");
        nota = scanner.nextFloat();
        if(nota >= 6) {
            System.out.println("Aprovado");
        } else {
            System.out.println("Reprovado");
        }
    }
}